virtualization/windowscontainers/container-networking/network-isolation-security.md

```diff
@@ -24,26 +24,47 @@ Each container endpoint is placed in its own __network compartment__ which is an
 Get-NetCompartment
 ```
 
-## Windows Firewall Security
-
-The Windows Firewall is used to enforce network security through port ACLs.
+## Network Security
+Depending on which container and network driver is used, Windows Firewall or VFP is used to enforce network security through port ACLs.
 
 > NOTE: Make sure your environment satisfies these *required* [prerequisites](https://docs.docker.com/network/overlay/#operations-for-all-overlay-networks) for creating overlay networks.
 
 
 ### Windows Server containers
-These use the Windows hosts' firewall (enlightened with network compartments.) 
+These use the Windows hosts' firewall (enlightened with network namespaces)
   * Default Outbound: ALLOW ALL
-  * Default Inbound: DENY ALL unsolicited
-    * Except "common" (e.g. DHCP, DNS, ICMP, etc.) network traffic
+  * Default Inbound: ALLOW ALL "common" (e.g. TCP,DHCP, DNS, ICMP, etc.) unsolicited network traffic
+    * DENY ALL "uncommon" network traffic
     * Create inbound ALLOW rule in response to ``docker run -p`` (port forwarding)
 
+
 ### Hyper-V containers
 Hyper-V containers have their own isolated kernel and hence run their own instance of Windows Firewall with the following configuration:
   * Default ALLOW ALL in Windows Firewall
 
 Note that in general, all container endpoints attached to an overlay network have an:
-  *  ALLOW ALL rule created.  
+  *  ALLOW ALL rule created.
 
 
 ![text](media/windows-firewall-containers.png)
+
+
+### Kubernetes Pods
+In [Kubernetes pods](https://kubernetes.io/docs/concepts/workloads/pods/pod/), an infrastructure container is first created to which an endpoint is attached. Containers (including infrastructure and worker containers) belonging to the same pod share a common network namespace (same IP and port space).
+
+
+![text](media/pod-network-compartment.png)
+
+
+### Customizing default port ACLs
+If you wish to modify the default port ACLs, please reference our HNS documentation (link to be added very soon). You will need to update policies inside the following components:
```

Comment:
I like the table below. I think I would mention here that ACLs are enforced by a combination of the Windows Firewall and VFP. Then the table makes more sense.